tests/test_features_integration.py

```diff
@@ -322,3 +322,41 @@ def test_catch22_all_features(dummy_data):
 
     res_df = feature_collection.calculate(dummy_data.first("15min"), return_df=True)
     assert (res_df.shape[0] > 0) and (res_df.shape[1]) > 0
+
+
+## ANTROPY
+
+
+def test_antropy_all_features(dummy_data):
+    # Tests if we integrate with ALL antropy features
+    # -> this requires no additional wrapper!
+    import antropy as ant
+
+    funcwrapper_entropy_funcs = [
```

Comment:
This is already rather niche; when we need a func-wrapper; as such I would include this somewhere in an example noteobok

Reply:
Sure! However, we need a funcwrapper quite often.

Guess we should make an example notebook for the following:
- pycatch22
- antropy

Reply:
indeed!

Reply:
Hmm as only a funcwrapper is necessary for antropy, I would rather just add this to the readme in examples than creating a full-blown notebook for it?

Reply:
Eventually ended up making an example notebook :upside_down_face: